de

```diff
@@ -1,5 +1,11 @@
 #!/usr/bin/env bash
 
+if [ ! -f ~/.de ]; then
+  echo "Error! File not found!"
```

Comment:
In order to make sure that this is shown as an error, I would recommend printing it out as a `fail` by using the functions that are defined underneath.

Reply:
This 1) makes it red and 2) sends it to stderr so it is recognizable by the computer as an error.